test/chrome-launcher-test.ts

```diff
@@ -64,6 +64,37 @@ describe('Launcher', () => {
     assert.strictEqual(fs.rmdir.callCount, 0);
   });
 
+  it('allows to overwrite browser prefs', async () => {
+    const existStub = stub().returns(true)
+    const readFileStub = stub().returns(Buffer.from(JSON.stringify({ some: 'prefs' })))
```

Comment:
will probably need to adjust these stubs to match readFile behavior (string, not buffer)